pypeit/core/findobj_skymask.py

```diff
@@ -1143,11 +1143,18 @@ def objs_in_slit(image, ivar, thismask, slit_left, slit_righ, inmask=None, fwhm=
 
     #sky_rect = np.repeat(sky_median[:, np.newaxis], nsamp, axis=1)
     #sky_rect = 0.0*image_rect
+
+
+    # Apply find_min_max_out
+    gpm_sigclip = np.ones_like(image_rect, dtype=bool)
     # Smash out the spectral direction masking outlying pixels. We use this mask gpm_sigclip below
-    data = np.ma.MaskedArray(image_rect, mask=np.logical_not(gpm_rect))
+    data = np.ma.MaskedArray(image_rect[find_min_max_out[0]:find_min_max_out[1],:], 
```

Comment:
I agree with what you are doing here, but it seems cleaner to code it as follows:

find_min_max_gpm = np.zeros_like(image_rect, dtype=bool)
find_min_max_gpm[find_min_max_out[0]: find_min_max_out[1], :] = True
# the total gpm = gpm_rect & find_min_max_gpm
data = np.ma.MaskedArray(image_rect, mask=np.logical_not(gpm_rect & find_min_max_gpm)
sigclip = stats.SigmaClip(sigma=sigclip_smash, maxiters=25, cenfunc='median', stdfunc=utils.nan_mad_std)
data_clipped, lower, upper = sigclip(data, axis=0, masked=True, return_bounds=True)
gpm_sigclip = np.logical_not(data_clipped.mask) 

Do you agree?

Reply:
yes, that is sensible.   Using this now.